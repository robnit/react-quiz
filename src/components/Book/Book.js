import React from 'react'

const Book = () => (
  <h1>BOOKS</h1>
)

export default Book