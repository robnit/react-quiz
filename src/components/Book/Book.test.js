import React from 'react'
import Book from 'components/Books/Books'
import { shallow } from 'enzyme' 

describe('book', () => {
  it('should match snapshots', () => {
    const wrapper = shallow(<Book/>)
   
    expect(wrapper).toMatchSnapshot()
  })
})
