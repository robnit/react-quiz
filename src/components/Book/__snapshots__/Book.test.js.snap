// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`book should match snapshots 1`] = `
<h1>
  BOOKS
</h1>
`;
